spec/services/quarterly_message_integration_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "business_time"
+
+# I really hate putting this test here as it does not correspond
```

Comment:
yes, I felt like it sort of would fit into DailyMessageSender also.  Or, we can just delete it altogether; sometimes when things don't fit, it means...they don't fit.  :)